### DISCUSSION TITLE
Creating a smart playlist of albums released in the past 3 years

### DISCUSSION THREAD

**Discussion by dmonder** [beetbox/beets#5125] [Q&A]
I am trying to figure out how to create a smart playlist that includes all the albums that have been released in the past 3 years. I want this to be dynamic so that it will always have the most recent albums. 

I have tried the following in the query field, but neither worked:

`year:{$py3:print(datetime.datetime.date.today().year - 3)}..`

`year:$(date +%Y --date="3 years ago")..`

**Comment by arsaboo** [comment]
Try `year:-3y..`

**Reply by dmonder** [reply]
Sorry. I tried that as well. It gives the same error as the other two.
